Cannot find name 'RequestInfo'. Did you mean 'RequestInit'

Any idea why I am getting this error when running tsc:

```
node_modules/hono/dist/hono.d.ts:47:22 - error TS2552: Cannot find name 'RequestInfo'. Did you mean 'RequestInit'?

47     request: (input: RequestInfo, requestInit?: RequestInit) => Promise<Response>;
```

Config:

```
{
  "compilerOptions": {
    "allowJs": true,
    "target": "esnext",
    "lib": ["esnext"],
    "moduleResolution": "node",
    "resolveJsonModule": true,
    "inlineSourceMap": true,
    "module": "esnext",
    "esModuleInterop": true,
    "experimentalDecorators": true,
    "emitDecoratorMetadata": true,
    "strict": true,
    "noImplicitAny": true,
    "noEmit": true,
    "types": [
      "@cloudflare/workers-types"
    ]
  },
  "ts-node": {
    "transpileOnly": true
  },
  "include": ["./src/**/*"]
}

```

--- a/deno_dist/hono.ts
+++ b/deno_dist/hono.ts
@@ -243,7 +243,7 @@ export class Hono<
     return this.dispatch(request, executionCtx, Environment)
   }
 
-  request = async (input: RequestInfo, requestInit?: RequestInit) => {
+  request = async (input: Request | string, requestInit?: RequestInit) => {
     const req = input instanceof Request ? input : new Request(input, requestInit)
     return await this.fetch(req)
   }

--- a/deno_dist/request.ts
+++ b/deno_dist/request.ts
@@ -38,7 +38,7 @@ declare global {
     parseBody<BodyType extends BodyData>(): Promise<BodyType>
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     jsonData?: any
-    json<JSONData = unknown>(): Promise<Partial<JSONData>>
+    json<T>(): Promise<T>
     data: Data
     valid: {
       (key: string | string[], value: unknown): Data

--- a/src/hono.ts
+++ b/src/hono.ts
@@ -243,7 +243,7 @@ export class Hono<
     return this.dispatch(request, executionCtx, Environment)
   }
 
-  request = async (input: RequestInfo, requestInit?: RequestInit) => {
+  request = async (input: Request | string, requestInit?: RequestInit) => {
     const req = input instanceof Request ? input : new Request(input, requestInit)
     return await this.fetch(req)
   }

--- a/src/request.ts
+++ b/src/request.ts
@@ -38,7 +38,7 @@ declare global {
     parseBody<BodyType extends BodyData>(): Promise<BodyType>
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     jsonData?: any
-    json<JSONData = unknown>(): Promise<Partial<JSONData>>
+    json<T>(): Promise<T>
     data: Data
     valid: {
       (key: string | string[], value: unknown): Data

